tests/runtime/variable

```diff
@@ -49,6 +49,7 @@ NAME 32-bit tracepoint arg
 PROG tracepoint:random:urandom_read { $i = args->got_bits; printf("bits: %d\n", $i); if ($i == 24) { exit() } }
 EXPECT bits: 24
 TIMEOUT 5
+MAX_KERNEL 5.17
```

Comment:
If `tracepoint:random:urandom_read` is not available since 5.17, then this should be set to 5.16, shouldn't it?

Reply:
If the current kernel is 5.17.0-rc5, Then

 LooseVersion("5.17.0-rc5") > LooseVersion("5.17") returns True. 

This means kernel 5.17.0-rc4 wouldnt be ignored. However, kernel 5.17.0-rc6 would fail.

Reply:
Added the suggestions